api/app/signals/apps/api/v1/validation/mixin.py

```diff
@@ -17,4 +17,9 @@ def validate(self, attrs):
                     reporter_email.endswith(settings.API_TRANSFORM_SOURCE_BASED_ON_REPORTER_DOMAIN_EXTENSIONS)):
                 attrs['source'] = settings.API_TRANSFORM_SOURCE_BASED_ON_REPORTER_SOURCE
 
+        if (self.feature_enabled('API_TRANSFORM_SOURCE_IF_A_SIGNAL_IS_A_CHILD')
+                and hasattr(settings, 'API_TRANSFORM_SOURCE_BASED_ON_SIGNAL_IS_A_CHILD')
+                and 'parent' in attrs and attrs['parent']):
```

Comment:
This code is active also for POSTs to the public signals endpoint, but will never be run because the `parent` field cannot be set in the `PublicSignalCreateSerializer` --- right?

Reply:
True